### ISSUE TITLE
[reorderable]=false not respected

### ISSUE THREAD

**Issue by sjonany** [swimlane/ngx-datatable#387]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here
```

**Current behavior**
   <ngx-datatable
....
      [reorderable]=false>
      >
The columns are still reorderable

**Expected behavior**
The columns should not be reorderable

* **More details**
Searching in this repository for "reorderable", I don't actually see this variable being used anywhere in src/components/datatable.component.ts, whereas other options like "scrollbarV" get used elsewhere in the file.

* **Table version:** 4.0.0
* **Angular version:** 2.4.0
* **Browser:** Chrome
* **Language:** Typescript 2.0.10

**Comment by florent1933** [comment]
Same problem, it's always reorderable.

**Comment by amcdnl** [comment]
Fixed in 4.1
